airbyte-integrations/bases/base-normalization/normalization/transform_catalog/stream_processor.py

```diff
@@ -493,29 +539,41 @@ def generate_scd_type_2_model(self, from_table: str, column_names: Dict[str, Tup
   {%- for field in fields %}
     {{ field }},
   {%- endfor %}
-    {{ cursor_field }} as _airbyte_start_at,
-    lag({{ cursor_field }}) over (
-        partition by {{ primary_key }}
-        order by {{ cursor_field }} is null asc, {{ cursor_field }} desc, _airbyte_emitted_at desc
-    ) as _airbyte_end_at,
-    lag({{ cursor_field }}) over (
-        partition by {{ primary_key }}
-        order by {{ cursor_field }} is null asc, {{ cursor_field }} desc, _airbyte_emitted_at desc{{ cdc_updated_at_order }}
-    ) is null {{ cdc_active_row }}as _airbyte_active_row,
-    _airbyte_emitted_at,
-    {{ hash_id }}
+  {{ cursor_field }} as {{ airbyte_start_at }},
+  lag({{ cursor_field }}) over (
+    partition by {{ primary_key }}
+    order by {{ cursor_field }} {{ order_null }}, {{ cursor_field }} desc, {{ col_emitted_at }} desc
+  ) as {{ airbyte_end_at }},
+  case when lag({{ cursor_field }}) over (
+    partition by {{ primary_key }}
+    order by {{ cursor_field }} {{ order_null }}, {{ cursor_field }} desc, {{ col_emitted_at }} desc{{ cdc_updated_at_order }}
+  ) is null {{ cdc_active_row }} then 1 else 0 end as {{ active_row }},
+  {{ col_emitted_at }},
+  {{ hash_id }}
```

Comment:
I changed the function to use `case when` so this works with Oracle and others destinations.

Reply:
Ok, that seems to work... 
if you think that "active_row" as an `Integer` is better than a `Boolean`...

Reply:
I prefer boolean type, but... https://stackoverflow.com/questions/14731971/create-boolean-attribute-in-oracle.

`create table example(b boolean);`

```
Error starting at line : 1 in command -
create table example(b boolean)
Error report -
ORA-00902: invalid datatype
00902. 00000 -  "invalid datatype"
*Cause:    
*Action:

```